packages/gatsby/src/services/create-pages.ts

```diff
@@ -0,0 +1,67 @@
+import { IBuildContext } from "./"
+
+import reporter from "gatsby-cli/lib/reporter"
+import apiRunnerNode from "../utils/api-runner-node"
+
+export async function createPages({
+  parentSpan,
+  gatsbyNodeGraphQLFunction,
+  store,
+}: Partial<IBuildContext>): Promise<void> {
+  if (!store) {
+    reporter.panic(`store not initialized`)
+  }
+  const activity = reporter.activityTimer(`createPages`, {
+    parentSpan,
+  })
+  activity.start()
+  // const timestamp = Date.now()
+  // const currentPages = new Map<string, IGatsbyPage>(store.getState().pages)
```

Comment:
Did you leave these intentionally? Same for comments further down

Reply:
Yes, because these are temporarily removed just while we're handling this outside the state machine. The main state machine branch has that code it, and I want to make it obvious where it's meant to go back in.